core/app/models/spree/image/active_storage_attachment.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'active_storage'
+
+module Spree::Image::ActiveStorageAttachment
```

Comment:
I can do that, but used this way there's no risk of incurring in the problem described in #2572 (at least I don't see it). The problem there was using the `class` keyword on an autoloaded constant with different superclass declaration.

Sometimes it was `class A < B`, sometimes `class A`, depending on load-order if the second form was loaded first it would result in an error (the other way around works because you can omit the superclass when reopening a class).

Let me know if the change is still needed 👍

Reply:
What if this module is someway loaded before `Spree::Image`?

Reply:
It will autoload `Spree::Image` and then if it's the configured module will go back and load itself. 
That said I tried to load it by itself but I didn't succeed, so I don't think that's an issue, do you have any specific case in mind?

Reply:
No, I think it can work but I'd still change it for consistency. I also found this related PR https://github.com/solidusio/solidus/pull/2098 and I think the goal there was to explicitly inform the type of all the modules/classes involved in the inheritance.